src/main/java/com/monumental/controllers/MonumentController.java

```diff
@@ -21,4 +23,15 @@ public Monument createMonument(Monument monument) {
     public Monument getMonument(@PathVariable("id") Integer id) {
         return this.monumentService.get(id);
     }
+
+    @GetMapping("/api/monuments")
+    public List<Monument> getAllMonuments() {
+        return this.monumentService.getAll();
+    }
+
+    @PutMapping("/api/monument/{id}")
```

Comment:
`@PutMapping` instead of `@PostMapping`?

Are we using PUT for updates and POST for creation?